Index: /branches/VBox-7.0/src/VBox/Devices/Network/SrvIntNetR0.cpp
===================================================================
--- a/src/VBox/Devices/Network/SrvIntNetR0.cpp	(revision 162842)
+++ b/src/VBox/Devices/Network/SrvIntNetR0.cpp	(revision 162843)
@@ -2936,5 +2936,5 @@
     union
     {
-        uint8_t     abBuf[sizeof(INTNETSG) + sizeof(INTNETSEG)];
+        uint8_t     abBuf[sizeof(INTNETSG) + 2 * sizeof(INTNETSEG)];
         INTNETSG    SG;
     } u;
