Description: Fix FTBFS with X Server 1.12.
Author: Felix Geyer <debfx-pkg@fobos.de>
Bug-Debian: http://bugs.debian.org/671061

--- a/src/VBox/Additions/common/crOpenGL/fakedri_drv.c
+++ b/src/VBox/Additions/common/crOpenGL/fakedri_drv.c
@@ -29,6 +29,7 @@
 #include <dlfcn.h>
 #include <elf.h>
 #include <unistd.h>
+#include "xorg-server.h"
 /** X server message type definitions. */
 typedef enum {
     X_PROBED,			/* Value was probed */
